invenio_records_resources/services/errors.py

```diff
@@ -20,6 +20,12 @@ class PermissionDeniedError(PermissionDenied):
 
     description = "Permission denied."
 
+    def __init__(self, action_name=None, record=None, *args, **kwargs):
```

Comment:
due to modification done to the constructor signature this is breaking change 🔴 
are we sure this is the only way of passing record context to the error handler?